packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -71,17 +160,24 @@ export class GuxAvatar {
 
   render(): JSX.Element {
     return (
-      <div
-        class={{
-          'gux-avatar': true,
-          [`gux-${this.size}`]: true,
-          [`gux-accent-${this.getAccent()}`]: true
-        }}
-      >
-        <slot name="image">
-          <abbr title={this.name}>{this.generateInitials()}</abbr>
-        </slot>
-      </div>
+      <Fragment>
```

Comment:
Is there a reason for this change to a wrapping fragment? I can't tell.

Reply:
You cant have 2 children there .gux-avatar & {this.renderBadge()} with JSX. Fragment wraps them them in JSX so you don't need a div as a wrapper that doesn't do anything or has no semantic meaning.

Reply:
You can also return an array in that case. Fragment has special uses like Host iirc.
`
[
        <div
          class={{
            'gux-avatar': true,
            [`${this.presence}`]: true,
            [`gux-${this.size}`]: true,
            'gux-presence-ring': this.presenceRing,
            [`gux-accent-${this.getAccent()}`]: true
          }}
        >
          <div class="gux-content">
            <slot name="image">
              <abbr title={this.name}>{this.generateInitials()}</abbr>
            </slot>
          </div>
        </div>,
        {this.renderBadge()}
] as JSX.Element;
`

Reply:
I think that's a good suggestion. Applied